test/uk/gov/hmrc/nisp/controllers/FullNIRecordControllerSpec.scala

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright 2016 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.nisp.controllers
```

Comment:
Rather than overriding the default flag, we thought it would be better if we had a separate spec, that encompasses testing the NI record page with the config flag as true. This way we test in full both scenarios independently. 
It is a duplicate of the NIControllerSpec with one addition which  tests for a value that only exists if the flag is true.